en/08_Changelogs/6.0.0.md

```diff
@@ -113,49 +115,36 @@ Email [community@silverstripe.org](mailto:community@silverstripe.org) if you are
 |[silverstripe/webauthn-authenticator](https://github.com/silverstripe/silverstripe-webauthn-authenticator)|CMS 6 compatible without commercial support|
 |[symbiote/silverstripe-multivaluefield](https://github.com/symbiote/silverstripe-multivaluefield)|Dropped|
 
-The code in `silverstripe/externallinks`, `silverstripe/security-report`, and `silverstripe/sitewidecontent-report` has been copied into `silverstripe/reports` and will be maintained there going forward. The namespaces for classes in those modules has been updated to `SilverStripe\Reports`. Note that any code that related to `silverstripe/subsite` or `silverstripe/contentreview` integration has been removed.
+The code in `silverstripe/externallinks`, `silverstripe/security-report`, and `silverstripe/sitewidecontent-report` has been copied into `silverstripe/reports` and will be maintained there going forward. The namespaces for classes in those modules has been updated to `SilverStripe\Reports`. Note that any code that related to `silverstripe/subsites` or `silverstripe/contentreview` integration has been removed.
 
 CMS 6 compatible versions of `silverstripe/blog`, `silverstripe/crontask`, `silverstripe/hybridsessions`, `silverstripe/subsites`, and `silverstripe/webauthn-authenticator` have been released with CMS 6.0.0, though they are not commercially supported.
 
 ### `silverstripe/campaign-admin` no longer part of `silverstripe/recipe-cms` {#campaign-admin}
 
-The `silverstripe/campaign-admin` module is no longer included in `silverstripe/recipe-cms`. If your `composer.json` requires `silverstripe/recipe-cms` and you wish to continue using `silverstripe/campaign-admin`, then you will need to add manually add to your project's `composer.json`.
+The `silverstripe/campaign-admin` module is no longer included in `silverstripe/recipe-cms`. If your `composer.json` requires `silverstripe/recipe-cms` and you wish to continue using `silverstripe/campaign-admin`, then you will need to add manually add it to your project's `composer.json`.
 
 ### New supported modules
 
 Some new modules have been created and are now covered by our [supported modules policy](/project_governance/supported_modules/). These modules will be supported for the full lifetime of Silverstripe CMS 6.
 
-> [!WARNING]
-> To continue using TinyMCE as an HTML editor in your existing project, you must add `silverstripe/htmleditor-tinymce` as an explicit dependency.
```

Comment:
This looked weird now that the "TinyMCE is in a separate module" section is so close to this - you could see the same warning twice in the same viewport:

![Screenshot from 2025-06-05 14-04-19](https://github.com/user-attachments/assets/9723af62-1d52-4565-a990-53834180a07a)

So instead, I've removed this warning but moved the TinyMCE section up:

![Screenshot from 2025-06-05 14-04-29](https://github.com/user-attachments/assets/ffc0874c-fd38-4b21-8655-1c583b31d4ef)